codice/ddf
# DDF-5041 Update Ace to Latest Version

#### What does this PR do?
Updates Ace to Latest version which fixes Windows builds
Renames .ts typescript files to .tsx so that line endings are handled properly
#### Who is reviewing it? 
@hayleynorton 
@zta6 
@Corey-Collins 
#### Ask 2 committers to review/merge the PR and tag them here
@adimka 
@djblue 
#### How should this be tested?
Ensure build passes on windows
Ensure build passes on mac
Remove license header from a file in the ui directory and verify that `yarn format` fails
#### Any background context you want to provide?
#### What are the relevant tickets?
Fixes: #5041 

#### Screenshots

#### Checklist:

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.
